### PR TITLE
Update props for SEO component

### PR DESCRIPTION
**Pull request by kubahasek**

### CODE REVIEW

File: components/basecomponents/SEO.tsx [hunk 1]
```diff
@@ -3,25 +3,26 @@ import Head from 'next/head';
...
 
-const SEO = ({ title, description, image }: Props) => {
+const SEO = ({ title, description, image = 'https://www.movapp.cz/icons/movapp-cover.jpg', noreferrer = true }: Props) => {
   return (
     <Head>
-      <meta name="referrer" content="no-referrer" />
+      {noreferrer && <meta name="referrer" content="no-referrer" />}
       <title>{title}</title>
-      <meta name="description" content={description} />
+      <meta name="description" content={description ? description : title} />
```
Comment (JuroUhlar) [inline components/basecomponents/SEO.tsx:16]: Try this instead: `description ?? title` :) 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Comment (JuroUhlar) [reply components/basecomponents/SEO.tsx:16]: Also, just assign it to a variable and re-use it: 

`const appliedDescription = description ?? title;`

and then 

` <meta name="og:description" content={appliedDescription} />`

**Review by JuroUhlar — CHANGES_REQUESTED** [review]

File: components/basecomponents/SEO.tsx [hunk 2]
```diff
@@ -3,25 +3,26 @@ import Head from 'next/head';
...
   title: string;
-  description: string;
-  image: string;
+  description?: string;
+  image?: string;
+  noreferrer?: boolean;
 }
 
-const SEO = ({ title, description, image }: Props) => {
+const SEO = ({ title, description, image = 'https://www.movapp.cz/icons/movapp-cover.jpg', noreferrer = true }: Props) => {
```
Comment (JuroUhlar) [inline components/basecomponents/SEO.tsx:11]: You can also see all uses of `SEO` and remove the `image` prop if it contains the default value :)

**Comment by JuroUhlar** [comment]
@kubahasek Nice, can you resolve the merge conflict please so we can merge it? Thanks a lot! :)

**Comment by kubahasek** [comment]
@JuroUhlar Conflict fixed! :)

**Review by JuroUhlar — APPROVED** [review]

**Pull request merged**